src/index.js

```diff
@@ -6,40 +10,47 @@ import Archivist from './archivist/index.js';
 import GitHub from './github/index.js';
 import logger from './logger/index.js';
 import Notifier from './notifier/index.js';
+import GitAdapter from './storage-adapters/git/index.js';
+import MongoAdapter from './storage-adapters/mongo/index.js';
 
 const args = process.argv.slice(2);
 const refilterOnly = args.includes('--refilter-only');
 const schedule = args.includes('--schedule');
+const extraArgs = args.filter(arg => !arg.startsWith('--'));
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
 (async function startOpenTermsArchive() {
-  const archivist = new Archivist();
+  const { versionsStorageAdapter, snapshotsStorageAdapter } = initStorageAdapters();
+
+  const archivist = new Archivist({
+    storage: {
+      versions: versionsStorageAdapter,
+      snapshots: snapshotsStorageAdapter,
+    },
+  });
 
   archivist.attach(logger);
+
   await archivist.init();
 
   logger.info('Start Open Terms Archive\n');
 
-  let serviceIds = args.filter(arg => !arg.startsWith('--'));
+  let serviceIds;
 
-  serviceIds = serviceIds.filter(serviceId => {
-    const isServiceDeclared = archivist.serviceDeclarations[serviceId];
+  if (extraArgs.length) {
+    serviceIds = extraArgs.filter(serviceId => {
+      const isServiceDeclared = archivist.serviceDeclarations[serviceId];
 
-    if (!isServiceDeclared) {
-      logger.warn(`Service ${serviceId} does not exist and will be ignored.`);
-    }
+      if (!isServiceDeclared) {
+        logger.warn(`Service ${serviceId} does not exist and will be ignored.`);
```

Comment:
```suggestion
        logger.warn(`Parameter "${serviceId}" was interpreted as a service ID to update, but no matching declaration was found. It will be ignored.`);
```